src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static Task[] tasks = new Task[100];
```

Comment:
Please avoid using magic number literals. You can use a constant and add meaningful names.